pandas/tools/tests/test_concat.py

```diff
@@ -1666,6 +1666,39 @@ def test_union_categoricals_ordered(self):
         with tm.assertRaisesRegexp(TypeError, msg):
             union_categoricals([c1, c2])
 
+    def test_union_categoricals_ignore_order(self):
+        c1 = Categorical([1, 2, 3], ordered=True)
+        c2 = Categorical([1, 2, 3], ordered=False)
+
+        res = union_categoricals([c1, c2], ignore_order=True)
+        exp = Categorical([1, 2, 3, 1, 2, 3])
+        tm.assert_categorical_equal(res, exp)
+
+        res = union_categoricals([c1, c1], ignore_order=True)
+        exp = Categorical([1, 2, 3, 1, 2, 3], ordered=False)
```

Comment:
This was intended to test two ordered categoricals with identical categories and orders.  The above was a mixed ordered and unordered with identical categories.

I did remove ordered=False from this test and other tests since it is the default.